consensus/spos/bls/subroundStartRound_test.go

```diff
@@ -809,6 +820,353 @@ func TestSubroundStartRound_InitCurrentRoundShouldMetrics(t *testing.T) {
 	})
 }
 
+func buildDefaultSubround(container spos.ConsensusCoreHandler) *spos.Subround {
+	ch := make(chan bool, 1)
+	consensusState := initConsensusState()
+	sr, _ := spos.NewSubround(
+		-1,
+		bls.SrStartRound,
+		bls.SrBlock,
+		int64(85*roundTimeDuration/100),
+		int64(95*roundTimeDuration/100),
+		"(START_ROUND)",
+		consensusState,
+		ch,
+		executeStoredMessages,
+		container,
+		chainID,
+		currentPid,
+		&statusHandler.AppStatusHandlerStub{},
+	)
+
+	return sr
+}
+
+// This test should return ErrNilHeader when GetGenesisHeader is called by Blockchain component
```

Comment:
can you remove the comments on the tests?
the tests are clear enough.

Reply:
updated